### PR TITLE
bokken: fix deps & bottle

### LINKED ISSUE #50014: bokken needs to depend on librsvg
bokken 1.8's formula does not depend on librsvg at the moment, but when running the program it yields this error:

```bash
agatti@tardigrade ~ $ bokken
	Python version...	OK
	Radare availability...	OK
2016-03-11 14:34:05.944 Python[79365:922309] *** WARNING: Method userSpaceScaleFactor in class NSView is deprecated on 10.7 and later. It should not be used in new applications. Use convertRectToBacking: instead.
	GTK UI dependencies...	OK
	GtkSourceView2...	OK
	Graphviz binaries...	OK
Starting bokken 1.8, running on:
  Python version:
    2.7.11 (default, Dec  6 2015, 16:00:09)
    [GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)]
  GTK version: 2.24.30
  PyGTK version: 2.24.0

Traceback (most recent call last):
  File "/usr/local/Cellar/bokken/1.8/libexec/bokken/bokken.py", line 76, in <module>
    bokken()
  File "/usr/local/Cellar/bokken/1.8/libexec/bokken/bokken.py", line 71, in bokken
    'radare' if args.radare else '',
  File "/usr/local/Cellar/bokken/1.8/libexec/bokken/ui/main.py", line 395, in main
    BokkenGTKClient(target, backend)
  File "/usr/local/Cellar/bokken/1.8/libexec/bokken/ui/main.py", line 96, in __init__
    dialog = file_dialog.FileDialog(self.target, True)
  File "/usr/local/Cellar/bokken/1.8/libexec/bokken/ui/file_dialog.py", line 41, in __init__
    ui.gtk2.common.set_bokken_icon(self)
  File "/usr/local/Cellar/bokken/1.8/libexec/bokken/ui/gtk2/common.py", line 42, in set_bokken_icon
    obj.set_icon_from_file(lib.common.datafile_path('bokken.svg'))
GError: Unable to load image-loading module: /usr/local/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: dlopen(/usr/local/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so, 1): image not found
```

however, after installing librsvg (and its dependency, libcroco), it runs normally:

```bash

agatti@tardigrade ~ $ brew install librsvg
==> Installing dependencies for librsvg: libcroco
==> Installing librsvg dependency: libcroco
==> Downloading https://homebrew.bintray.com/bottles/libcroco-0.6.11.el_capitan.
Already downloaded: /Library/Caches/Homebrew/libcroco-0.6.11.el_capitan.bottle.tar.gz
==> Pouring libcroco-0.6.11.el_capitan.bottle.tar.gz
🍺  /usr/local/Cellar/libcroco/0.6.11: 78 files, 1.7M
==> Installing librsvg
==> Downloading https://homebrew.bintray.com/bottles/librsvg-2.40.13.el_capitan.
Already downloaded: /Library/Caches/Homebrew/librsvg-2.40.13.el_capitan.bottle.tar.gz
==> Pouring librsvg-2.40.13.el_capitan.bottle.tar.gz
==> /usr/local/opt/gdk-pixbuf/bin/gdk-pixbuf-query-loaders --update-cache
🍺  /usr/local/Cellar/librsvg/2.40.13: 49 files, 1.6M
agatti@tardigrade ~ $ bokken
	Python version...	OK
	Radare availability...	OK
2016-03-11 14:34:37.382 Python[79557:923044] *** WARNING: Method userSpaceScaleFactor in class NSView is deprecated on 10.7 and later. It should not be used in new applications. Use convertRectToBacking: instead.
	GTK UI dependencies...	OK
	GtkSourceView2...	OK
	Graphviz binaries...	OK
Starting bokken 1.8, running on:
  Python version:
    2.7.11 (default, Dec  6 2015, 16:00:09)
    [GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)]
  GTK version: 2.24.30
  PyGTK version: 2.24.0
```

### PR DESCRIPTION
**Pull request by tdsmith**
Closes #50014.

### CODE REVIEW

**Comment by tdsmith** [comment]
Looks like upstream is down.

File: Library/Formula/bokken.rb [hunk 1]
```diff
@@ -32,6 +33,7 @@ def install
     resource("distorm64").stage do
       inreplace "src/pydistorm.h", "python2\.5", "python2.7"
       cd "build/mac" do
+        inreplace "Makefile", "-lpython", "-undefined dynamic_lookup"
```
Comment (mikemcquaid) [inline Library/Formula/bokken.rb]: Submitted upstream?

**Comment by DomT4** [comment]
You're green now, I added a mirror last night given the ongoing down-ness.

File: Library/Formula/bokken.rb (hunk 1 above)
Comment (tdsmith) [inline Library/Formula/bokken.rb]: No; I don't think this branch is maintained.

**Comment by tdsmith** [comment]
Bottle job is failing in merge? Tried rebasing...

**Comment by DomT4** [comment]
Passed this time. Not sure why it failed initially :confused:.

**Pull request closed without merging**